stdlib/REPL/src/LineEdit.jl

```diff
@@ -1577,7 +1578,7 @@ function keymap_merge(target::Dict{Char,Any}, source::Union{Dict{Char,Any},AnyDi
         if isa(value, Union{Char,String})
             value = getEntry(ret, value)
             if value === nothing
-                error("Could not find redirected value " * escape_string(source[key]))
+                error("Could not find redirected value " * escape_string(String(source[key])))
```

Comment:
Actually, this doesn't work, I think it hits the no inference on error path limitation.

Reply:
How about
```julia
newvalue = getEntry(ret, value)
if newvalue === nothing
    error("Could not find redirected value " * escape_string(String(value)))
end
value = newvalue
```